diff -Nur rpm-ostree-2018.8_old/Makefile.am rpm-ostree-2018.8/Makefile.am
--- rpm-ostree-2018.8_old/Makefile.am	2019-04-28 15:40:35.032000000 +0800
+++ rpm-ostree-2018.8/Makefile.am	2019-04-28 15:41:34.856000000 +0800
@@ -30,7 +30,7 @@
 SUBDIRS += api-doc
 endif
 
-RPM_OSTREE_GITREV=$(shell if command -v git >/dev/null 2>&1 && test -e $(srcdir)/.git; then git describe --abbrev=42 --tags --always HEAD; fi)
+RPM_OSTREE_GITREV=$(shell if command -v git >/dev/null 2>&1 && test -e $(srcdir)/.git; then echo 67cf76d85e7783b1f0007638bdad823998340338; fi)
 
 ACLOCAL_AMFLAGS += -I m4 ${ACLOCAL_FLAGS}
 # WITH_SWDB is a hackaround for it being used in libdnf headers
diff -Nur rpm-ostree-2018.8_old/rust/Cargo.toml rpm-ostree-2018.8/rust/Cargo.toml
--- rpm-ostree-2018.8_old/rust/Cargo.toml	2019-04-28 15:40:35.112000000 +0800
+++ rpm-ostree-2018.8/rust/Cargo.toml	2019-04-28 15:41:02.988000000 +0800
@@ -27,4 +27,4 @@
 
 [profile.release]
 panic = "abort"
-lto = true
+lto = false
